packages/dd-trace/src/appsec/iast/taint-tracking/plugin.js

```diff
@@ -170,6 +181,37 @@ class TaintTrackingPlugin extends SourceIastPlugin {
     this.taintHeaders(req.headers, iastContext)
     this.taintUrl(req, iastContext)
   }
+
+  _taintDatabaseResult (result, dbOrigin, iastContext = getIastContext(storage.getStore()), name) {
+    if (!iastContext) return result
+
+    let rowsToTaint = this.iastConfig?.dbRowsToTaint
+    if (typeof rowsToTaint !== 'number') {
```

Comment:
I think that we should trust `dbRowsToTaint` is a number since it's coming from config ?

Reply:
It is a simple and cheap check, why stop doing it? (and now it is done only once in `onConfigure`)